bootstrap.sh

```diff
@@ -1653,7 +1659,8 @@ function main {
   #   VENDOR= apple, redhat, centos, canonical
   #   VENDOR_RELEASE=
   #                  RHEL{rhel,Tikanga,Santiago}
-  #                  Ubuntu{ubuntu,Lucid,Maverick,Natty,Oneiric,Precise,Quantal}
+  #                  Ubuntu{ubuntu,Lucid,Maverick,Natty,Oneiric,Precise,Quantal,
+  #                         Raring,Trusty,Xenial}
```

Comment:
If we are adding Xenial in this commit should we also add it to the `$VENDOR_RELEASE` changes?